SPECS/cockpit/cockpit.spec

```diff
@@ -0,0 +1,87 @@
+Summary:        A package to provide cockpit for mariner
+Name:           cockpit
+Version:        248
+Release:        1%{?dist}
+License:        LGPLv2+, MIT
```

Comment:
>LGPLv2+, MIT [](http://example.com/codeflow?start=16&length=12)

If two or more licenses apply at the same time, they must be connected with `AND` (all-caps). If either one or another can be used, please use `OR`.